src/components/Main/Main.js

```diff
@@ -128,7 +132,9 @@ export class MainComponent {
     this.shareView = new ShareViewComponent(this.shareViewPoint);
 
     this.about = new AboutComponent(this.mainPoint);
-    this.table = new MySongsTableComponent(this.mainPoint);
+    this.table = new MySongsTableComponent(this.mainPoint, {
+      handlePlaySong: this.handlePlaySong.bind(this)
```

Comment:
the property should be called `onSongPlay`. and handler name should be `handleSongPlay`